Lesson4/step2/project.py

```diff
@@ -55,13 +56,13 @@ def fbconnect():
     h = httplib2.Http()
     result = h.request(url, 'GET')[1]
 
+
     # Use token to get user info from API
-    userinfo_url = "https://graph.facebook.com/v2.4/me"
+    userinfo_url = "https://graph.facebook.com/v2.8/me"
     # strip expire tag from access token
-    token = result.split("&")[0]
-
+    token = result.split(',')[0].split(':')[1].replace('"', '')
```

Comment:
This line could probably use an explanatory comment; the syntax being operated on here is non-obvious.